test_collections/sdk_tests/support/python_testing/models/python_test_parser.py

```diff
@@ -163,6 +163,18 @@ def __parse_test_case(
     if pics_method:
         tc_pics = __retrieve_pics(pics_method)
 
+    # - PythonTestType.COMMISSIONING: test cases that have a commissioning first step
+    # - PythonTestType.NO_COMMISSIONING: test cases that follow the expected template
+    #   but don't have a commissioning first step
+    # - PythonTestType.LEGACY: test cases that don't follow the expected template
+    # We use the desc_[test_name] method as an indicator that the test case follows the
+    # expected template
+    python_type = PythonTestType.LEGACY
```

Comment:
```suggestion
    python_test_type = PythonTestType.LEGACY
```
Maybe python_test_type has more meaning. What do you think?

Reply:
Done